rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+Since journeys may vary per persona, personas MUST be identified for an epic by assigning a documented GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
+
+#### Technical and Operational Requirements
+Product-wide technical or operational requirements are defined by **Technical Architecture** and modeled as epics in the *dcp* repo.
+
+#### Owners
+
+A DCP owner MUST be assigned to each issue or epic in the *Product Backlog* in the *dcp* repo to shepherd and coordinate progress. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to epics in the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own ZenHub repo and is linked to the relevant epic in the *dcp* repo.
+
+* No component-specific GitHub labels such as "Data Store" MUST be assigned to an epic in the *dcp* repo. The link documents the relationship.
+
+### Sprints
+
+ **Four week sprints**. There MUST be one DCP Sprint and Sprint Backlog, not a separate Sprint and Sprint Backlog for each component team. Each DCP Sprint starts on Thursday and ends on Wednesday.  
+ 
+Each Sprint MUST be assigned a ZenHub *Milestone* with a due date by the **Product Owners**.
+
+ All Scrum events are scheduled on the DCP calendar at a regular cadence. For example:
+![](../images/0000-scrum-calendar.png)
+
+### Refinement
+
+New items appear in the ZenHub *New* pipeline and must be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline.
+
+**NOTE**: *To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items SHOULD be limited to **DCP PM** or a facilitator from **Technical Architecture**.*
+
+The ZenHub *Product Backlog* pipeline is the DCP product backlog which is ordered during Refinement.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great constraint in the *dcp* repo. It's not a pipeline for avoiding hard decisions during Refinement. It's intended to document potential *futures* from the DCP roadmap. Individual component repo(s) MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+***QUESTION**: During the PM whiteboard session, there was agreement that dcp Epics would follow the normal pipeline lifecycle. Is this still true?*
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* pipeline. It will also be hidden in the dashboard. Individual component repo(s) MAY use the *Epic* pipeline in a manner of their own choosing.
```

Comment:
For clarity/readability, I would just state the desired process and eschew the historical bits.

Reply:
If you are referring to the ***QUESTION*** section, it was a temporary _look at this!_ note for the review by product owners. We agreed to one approach during the PM meeting, but I felt that there was potential _drift_ during later triage sessions. Removing now that there's confirmation of our original approach.

Reply:
I think you could also put questions on the PR itself. That's probably where the meta-discussion belongs.